lib/bn.js

```diff
@@ -53,6 +53,12 @@ else
 BN.BN = BN;
 BN.wordSize = 26;
 
+try {
+  BN._Buffer = require('buf' + 'fer').Buffer;
```

Comment:
Like I already mentioned in [PR to your other project](https://github.com/indutny/brorand/pull/3), webpack is (unfortunately) smart enough to break this hack.